src/schemas.py

```diff
@@ -26,3 +26,22 @@
         "repo_url",
     ],
 }
+
+worker_schema = {
```

Comment:
do we want to add this to the schemas in the README or hold off while its experimental?